src/Author.java

```diff
@@ -0,0 +1,25 @@
+public class Author {
+    String firstName;
+    String lastName;
```

Comment:
```suggestion
    private String firstName;
    private String lastName;
```
поля в классах должны быть приватные, про это написано даже в задании